tests/cypress/src/DashTable.ts

```diff
@@ -61,6 +61,30 @@ export default class DashTable {
         return cy.get(`#table tbody td.cell--selected`);
     }
 
+    static scrollToTop() {
+        cy.get(`.cell.cell-1-1.dash-fixed-content`).invoke(`outerHeight`).then(height => {
+            cy.scrollTo(0, -1 * height);
```

Comment:
Minor, but you can do this instead:

```suggestion
            cy.scrollTo(0, -height);
```